bin/seeds.js

```diff
@@ -24,6 +24,7 @@ for (let i = 0; i < 20; i++) {
 			username: Math.random().toString(36).substring(7),
 			userId: rndmId(),
 			testUser: true,
+			servers: [Math.random() > 0.3 ? "717462802272485516" : "8275918275289"]
```

Comment:
Hva er egentlig poenget med den her?

Reply:
717462802272485516 er vår server mens det andre tallet er bare noe random. Det er for å ha falske brukere i db som tilhører vår server og andre som er i en annen server.